radix-engine/src/system/kernel_modules/costing/fee_table.rs

```diff
@@ -88,26 +89,105 @@ impl FeeTable {
                     RESOURCE_MANAGER_CREATE_UUID_NON_FUNGIBLE_WITH_INITIAL_SUPPLY => {
                         self.fixed_high
                     }
+                    RESOURCE_MANAGER_MINT_NON_FUNGIBLE => self.fixed_high,
+                    RESOURCE_MANAGER_MINT_UUID_NON_FUNGIBLE => self.fixed_high,
+                    RESOURCE_MANAGER_MINT_FUNGIBLE => self.fixed_high,
+                    RESOURCE_MANAGER_BURN_BUCKET_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_BURN_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_CREATE_VAULT_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_CREATE_BUCKET_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_UPDATE_VAULT_AUTH_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_SET_VAULT_AUTH_MUTABILITY_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_UPDATE_NON_FUNGIBLE_DATA_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_NON_FUNGIBLE_EXISTS_IDENT => self.fixed_medium,
+                    RESOURCE_MANAGER_GET_TOTAL_SUPPLY_IDENT => self.fixed_low,
+                    RESOURCE_MANAGER_GET_NON_FUNGIBLE_IDENT => self.fixed_low,
+                    RESOURCE_MANAGER_GET_RESOURCE_TYPE_IDENT => self.fixed_low,
```

Comment:
Oh, you took the first approach for now.

Reply:
Yup, will update this in a future PR.